hrjobcontract/CRM/Hrjobcontract/Test/Fabricator/HRJobHour.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use CRM_Hrjobcontract_Test_Fabricator_BaseAPIFabricator as BaseAPIFabricator;
+
+class CRM_Hrjobcontract_Test_Fabricator_HRJobHour extends BaseAPIFabricator {
+
+  protected static function getEntityName() {
+    return 'HRJobHour';
```

Comment:
Just an idea: if the fabricators names are following the same format (`CRM_ExtensionName_Test_Fabricator_EntityName`), wouldn't it be possible to extract the entity name from the class name? This way you wouldn't need to redefine the `getEntityName()` implementation for every child class